gcloud/datastore/_implicit_environ.py

```diff
@@ -79,6 +86,16 @@ def compute_engine_id():
         connection.close()
 
 
+def _get_production_dataset_id():
```

Comment:
I'm not sure I buy the need for the one-liners. Even if we leave them, this one's name is confusing (I don't know what "production" has to do with its semantics).